src/Features/Core/Portable/Completion/Providers/SymbolCompletionItem.cs

```diff
@@ -50,6 +50,8 @@ internal static class SymbolCompletionItem
             }
 
             props = props.Add("ContextPosition", contextPosition.ToString());
+            props = WithSupportedPlatforms(props, supportedPlatforms);
```

Comment:
Given that we are now constructing the props first, we can use ImmutableDictionary.Builder to save a couple more allocations for each item creation

Reply:
Good idea!